src/resources/BaseInterfaces.ts

```diff
@@ -26,3 +26,16 @@ export interface PrivilegeModel {
     targetId: string;
     type?: string;
 }
+
+export enum SortingOrder {
+    ASC = 'ASC',
+    DESC = 'DESC',
+}
+
+export enum FieldTypes {
```

Comment:
We'll have to move these in the `Enums.ts` file once my PR is merged

Reply:
No problem~